src/model.ts

```diff
@@ -57,11 +58,17 @@ export function createLanguageModel(env: Record<string, string | undefined>): Ty
  * Creates a language model encapsulation of an OpenAI REST API endpoint.
  * @param apiKey The OpenAI API key.
  * @param model The model name.
+ * @param org: The OpenAI organization id.
  * @param endPoint The URL of the OpenAI REST API endpoint. Defaults to "https://api.openai.com/v1/chat/completions".
  * @returns An instance of `TypeChatLanguageModel`.
  */
-export function createOpenAILanguageModel(apiKey: string, model: string, endPoint = "https://api.openai.com/v1/chat/completions",): TypeChatLanguageModel {
-    return createAxiosLanguageModel(endPoint, { headers: { Authorization: `Bearer ${apiKey}` } }, { model });
+export function createOpenAILanguageModel(apiKey: string, model: string, org: string, endPoint = "https://api.openai.com/v1/chat/completions"): TypeChatLanguageModel {
```

Comment:
Welp, there's the first breaking change!

Maybe we should just switch these into object literals.

Reply:
Good point ... the endpoint was optional, the org was not, so I did I changed the order. Let me make it backward compat.